boxwise_flask/models/box.py

```diff
@@ -1,27 +1,77 @@
 import uuid
 from datetime import datetime
 
-from peewee import CharField, DateTimeField, IntegerField
+from peewee import (
+    SQL,
+    CharField,
+    DateTimeField,
+    ForeignKeyField,
+    IntegerField,
+    TextField,
+)
 
-from ..db import db
-from .qr import Qr
+from boxwise_flask.db import db
+from boxwise_flask.models.box_state import BoxState
+from boxwise_flask.models.location import Location
+from boxwise_flask.models.product import Product
+from boxwise_flask.models.size import Size
+from boxwise_flask.models.user import User
+
+from .qr_code import QRCode
 
 
 class Box(db.Model):
+    _gender = CharField(null=True)
+    _size = CharField(null=True)
+    _type = CharField(null=True)
+    box_id = CharField(constraints=[SQL("DEFAULT ''")], index=True)
+    box_state = ForeignKeyField(
+        column_name="box_state_id",
+        constraints=[SQL("DEFAULT 1")],
+        field="id",
+        model=BoxState,
+    )
+    comments = TextField()
+    created = DateTimeField(null=True)
+    created_by = ForeignKeyField(
+        column_name="created_by", field="id", model=User, null=True
+    )
+    deleted = DateTimeField(default=datetime.now())
```

Comment:
deleted fields should always include `null=true` and `default=null`.

Reply:
I had this as a question. I was confused why it wasn't already like this.